code/game/machinery/rechargestation.dm

```diff
@@ -28,26 +38,158 @@
 	for(var/obj/item/stock_parts/cell/C in component_parts)
 		recharge_speed *= C.maxcharge / 10000
 
+/obj/machinery/recharge_station/emag_act(mob/user)
+	if(obj_flags & EMAGGED)
+		return
+	obj_flags |= EMAGGED
+	playsound(src, "sparks", 75, TRUE, -1)
+	to_chat(user, span_notice("You use the cryptographic sequencer on [src] causing a well-hidden panel to open."))
```

Comment:
This implies that NT standard chargers are just meant to do this. Maybe reflavor to a hijacking of the borg repair gear inside?

Reply:
done